action/src/utils.ts

```diff
@@ -6,12 +6,18 @@ import { chmodSync } from 'fs';
 import { platform, arch } from 'os';
 import { normalize, join } from 'path';
 import { BranchSummary, SimpleGit, simpleGit } from 'simple-git';
+import { HttpClient, HttpClientResponse } from '@actions/http-client';
+import { OutgoingHttpHeaders } from 'http';
 
 export class Utils {
     private static readonly LATEST_RELEASE_VERSION: string = '[RELEASE]';
     private static readonly LATEST_CLI_VERSION_ARG: string = 'latest';
     private static readonly VERSION_ARG: string = 'version';
     private static readonly TOOL_NAME: string = 'frogbot';
+    // OpenID Connect audience input
+    private static readonly OIDC_AUDIENCE_ARG: string = 'oidc-audience';
+    // OpenID Connect provider_name input
+    private static readonly OIDC_INTEGRATION_PROVIDER_NAME: string = 'oidc-provider-name';
```

Comment:
I suggest being consistent with the naming convention - if it's an argument name - so both the OIDC args should end with _ARG